src/main/java/org/codeforamerica/shiba/output/FileDownLoadController.java

```diff
@@ -75,20 +87,71 @@ public FileDownLoadController(
             @PathVariable String applicationId,
             HttpServletRequest request
     ) {
-        String requestIpHeader = Optional.ofNullable(request.getHeader("X-FORWARDED-FOR")).orElse("");
-        String[] ipAddresses = requestIpHeader.split(",");
-        String requestIp = ipAddresses.length > 1 ? ipAddresses[ipAddresses.length - 2].trim() : "<blank>";
+        String requestIp = createRequestIp(request);
 
         applicationEventPublisher.publishEvent(new DownloadCafEvent(applicationId, requestIp));
         ApplicationFile applicationFile = pdfGenerator.generate(applicationId, CAF, CASEWORKER);
 
         return createResponse(applicationFile);
     }
 
+    @GetMapping("/download-docs/{applicationId}")
+    ResponseEntity<byte[]> downloadDocsWithApplicationId(
+            @PathVariable String applicationId,
+            HttpServletRequest request
+    ) throws IOException {
+        String requestIp = createRequestIp(request);
+
+        // TODO: Change this to a Doc download event
```

Comment:
Was this a change you wanted to make before merging?

Reply:
No we created another ticket to handle this